packages/sdk-node/package.json

```diff
@@ -24,34 +24,34 @@
   },
   "exports": {
     ".": {
-      "types": "./dist/types/index.d.ts",
-      "require": "./dist/cjs/index.js",
-      "import": "./dist/esm/index.mjs",
-      "default": "./dist/cjs/index.js"
+      "import": {
+        "types": "./dist/index.d.mts",
+        "default": "./dist/index.mjs"
+      },
+      "require": {
+        "types": "./dist/index.d.ts",
+        "default": "./dist/index.js"
+      }
     },
-    "./esm/instance": {
-      "types": "./dist/types/instance.d.ts",
-      "require": "./dist/cjs/instance.js",
-      "import": "./dist/esm/instance.mjs",
-      "default": "./dist/cjs/instance.js"
+    "./instance": {
+      "import": {
+        "types": "./dist/instance.d.mts",
+        "default": "./dist/instance.mjs"
+      },
+      "require": {
+        "types": "./dist/instance.d.ts",
+        "default": "./dist/instance.js"
+      }
     },
-    "./cjs/instance": {
-      "types": "./dist/types/instance.d.ts",
-      "require": "./dist/cjs/instance.js",
-      "import": "./dist/esm/instance.mjs",
-      "default": "./dist/cjs/instance.js"
-    }
+    "./package.json": "./package.json"
   },
-  "main": "./dist/cjs/index.js",
-  "module": "./dist/esm/index.mjs",
-  "types": "./dist/types/index.d.ts",
+  "main": "./dist/index.js",
   "files": [
-    "dist",
-    "esm",
-    "cjs"
+    "dist"
   ],
   "scripts": {
     "build": "npm run clean && tsup",
+    "postbuild": "node ../../scripts/subpath-workaround.mjs sdk-node",
```

Comment:
@dimkl probably i'm missing some context, but why do we need this ? feel free to point me to an internal document if necessary.

Reply:
This will generate a `packages/sdk-node/instance/` folder with a `package.json` to be used for subpath import of `@clerk/clerk-sdk-node/instance`. We have used it in `@clerk/shared`.